src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_vcf_parsing.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import vcf
```

Comment:
Ideally IMO we shouldn't be parsing anything in python except streams from java

Reply:
Let's call that a version 2.0 feature.